CONTRIBUTING.md

```diff
@@ -74,25 +74,27 @@ The test suite runs with `py.test`. The following are some useful commands for r
 tests, assuming you are in the `conda-build` root folder:
 
 ### Run all tests:
-```
-    py.test tests
+```bash
+    make test
```

Comment:
It might be good to add `# On Linux and Mac OS X` above the `make test` command, since using `make` is tricky on Windows